### PR TITLE
#169950175 bug(make-fix-ui): make fix ui

### PR DESCRIPTION
**Pull request by ebenndas**
[Finishes:#169950175]
> ### What does this PR do?
> - To make and fix UI be well designed

### CODE REVIEW

File: UI/css/style.css [hunk 1]
```diff
@@ -99,25 +99,29 @@ button{
...
      color: green;
-     background-color: whitesmoke;
 }
 
 footer {
      min-height: 10vh;
+     width: 100%;
      clear: both;
-     padding-top: 3%;
+     padding-top: 3%;
```
Comment (aaronsekisambu) [inline UI/css/style.css]: @ebenezeradukunda use padding in ``px` instead of `%`

File: UI/css/style.css [hunk 2]
```diff
@@ -128,21 +132,27 @@ footer {
...
      list-style: none;
-     color: sienna;
+     color: rgba(231, 172, 144, 0.89);
      cursor: pointer;
+     background: none;
 }
 
 footer nav ul li a {
      text-decoration: none;
+     background: none;
```
Comment (aaronsekisambu) [inline UI/css/style.css]: What are you trying to achieve here by `ul li a`? Nesting HTML element styling brings a lot of complexity. Please explain this.

File: UI/css/style.css [hunk 3]
```diff
@@ -548,26 +565,32 @@ tbody tr td ul li a{
           font-size: 14px;
      }
 
-     body {
+     * {
```
Comment (aaronsekisambu) [inline UI/css/style.css]: `*` everything please re-think this

**Review by aaronsekisambu — CHANGES_REQUESTED** [review]
@ebenezeradukunda Good work, work on those changes as you complete the PR

File: UI/css/style.css (hunk 1 above)
Comment (ebenndas) [reply UI/css/style.css]: Thanks for this, I have ready done it, it looks nice

Footer

File: UI/css/style.css (hunk 2 above)
Comment (ebenndas) [reply UI/css/style.css]: I wanted to specify the link which I have in footer

File: UI/css/style.css (hunk 3 above)
Comment (ebenndas) [reply UI/css/style.css]: Here I don't understand well

Comment (aaronsekisambu) [reply UI/css/style.css]: Where your using, `*` it means your styling the whole DOM with a wildcat/selection. remove it and style it accordingly with the right HTML elements

**Comment by aaronsekisambu** [comment]
@ebenezeradukunda you have issues when hovering you buttons, fix them before we merger the PR

**Review by aaronsekisambu — APPROVED** [review]

**Pull request merged**